scripts/clidocgen/main.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra/doc"
+
+	"github.com/coder/coder/cli"
+)
+
+type route struct {
+	Title       string  `json:"title,omitempty"`
+	Description string  `json:"description,omitempty"`
+	Path        string  `json:"path,omitempty"`
+	IconPath    string  `json:"icon_path,omitempty"`
+	State       string  `json:"state,omitempty"`
+	Children    []route `json:"children,omitempty"`
+}
+
+type manifest struct {
+	Versions []string `json:"versions,omitempty"`
+	Routes   []route  `json:"routes,omitempty"`
+}
+
+func main() {
+	cmd := cli.Root(cli.AGPL())
+
+	// Get paths
+	basePath := os.Getenv("BASE_PATH")
+	if basePath == "" {
+		log.Fatal("BASE_PATH should be defined")
+	}
+	markdownDocsDir := path.Join(basePath, "docs/cli")
+	manifestFilepath := path.Join(basePath, "docs/manifest.json")
+
+	// Generate markdown
+	err := doc.GenMarkdownTree(cmd, markdownDocsDir)
```

Comment:
In theory, you should remove the content of the CLI directory, before you generate new files.

Reply:
The `GenMarkdownTree` already does that, but do you think we should do it anyway?

Reply:
This is enough 👍  I wasn't aware that it clears the directory too.